src/components/ItemTable/ItemTableControls.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  Box,
+  Pagination,
+  ProgressIndicator,
+  Label,
+  Icon,
+} from "@nypl/design-system-react-components"
+import type { RefObject, SyntheticEvent } from "react"
+
+import { ITEM_PAGINATION_BATCH_SIZE } from "../../config/constants"
+import RCLink from "../Links/RCLink/RCLink"
+import type Bib from "../../models/Bib"
+
+interface ItemTableControlsProps {
+  bib: Bib
+  viewAllEnabled: boolean
+  itemsLoading: boolean
+  itemTablePage: number
+  handlePageChange: (selected: number) => void
+  handleViewAllClick: (e: SyntheticEvent) => Promise<void>
+  viewAllLoadingTextRef: RefObject<HTMLDivElement & HTMLLabelElement>
+}
+
+/**
+ * The ItemTableControls component appears below the Item table on the Bib page displays
+ * pagination and view all controls/messaging.
+ */
+const ItemTableControls = ({
+  bib,
+  viewAllEnabled,
+  itemsLoading,
+  itemTablePage,
+  handlePageChange,
+  handleViewAllClick,
+  viewAllLoadingTextRef,
+}: ItemTableControlsProps) => {
+  return (
+    <Box display="flex" my="xl" justifyContent="space-between">
+      {!viewAllEnabled ? (
+        <Pagination
+          id="bib-items-pagination"
+          initialPage={itemTablePage}
+          currentPage={itemTablePage}
+          pageCount={Math.ceil(
+            bib.numPhysicalItems / ITEM_PAGINATION_BATCH_SIZE
+          )}
+          onPageChange={handlePageChange}
+          width="auto"
+        />
+      ) : null}
+      {bib.showViewAllItemsLink &&
+        (itemsLoading && viewAllEnabled ? (
+          <Box
+            ml="auto"
+            display="flex"
+            justifyContent="flex-start"
+            alignItems="center"
+          >
+            <ProgressIndicator
+              id="bib-all-items-loading"
+              labelText={bib.itemsViewAllLoadingMessage}
+              size="small"
+              indicatorType="circular"
+              mr="xs"
+              isIndeterminate
+            />
+            <Label
```

Comment:
Create a ticket for this https://newyorkpubliclibrary.atlassian.net/browse/DSD-1798

But now I'm rethinking this and wondering if perhaps it's okay to leave the `ProgressIndicator` as is and replace `Label` with a span and remove its `htmlFor` attribute. Internally, the `ProgressIndicator` will add the labelText as an aria-label so this visible `Label` is redundant for screen readers.